cli/cmd/flags.go

```diff
@@ -24,6 +24,10 @@ func bindAVSAddress(cmd *cobra.Command, addr *string) {
 	cmd.Flags().StringVar(addr, "avs-address", *addr, "Optional address of the Omni AVS contract")
 }
 
+func bindDeveloperForgeProjectConfig(cmd *cobra.Command, cfg *developerForgeProjectConfig) {
+	cmd.Flags().StringVar(&cfg.templateURL, "template", "https://github.com/omni-network/hello-world-template.git", "URL of the forge template to use")
```

Comment:
This works - that way they can use any template they like. But we could also just use the name of it, and try to fetch `https://github.com/omni-network/{template}.git`